### PR TITLE
[MVC-986] Contact Page Fixes

### PR DESCRIPTION
**Pull request by SydneyMhashu**
Fix on the Addresses layout
Fixed the top banner alignment
Fixed test rendering for banner text and text alignment.
Borders for addresses

### CODE REVIEW

**Comment by SydneyMhashu** [comment]
Screenshot 2021-01-27 at 13 41 41
Screenshot 2021-01-27 at 13 42 09
Screenshot 2021-01-27 at 13 42 46
Screenshot 2021-01-27 at 13 43 23
Screenshot 2021-01-27 at 13 44 08
Screenshot 2021-01-27 at 13 44 29

File: contact.markdown [hunk 1]
```diff
@@ -16,11 +16,12 @@ Addresses:
...
 
     [hellolabs@triggerise.org](mailto:hellolabs@triggerise.org)
+AddressesRow2:    
 - Address: |-
     **Triggerise Ethiopia**
 
-    Gulf Aziz Building,
-    Office Number 120,
+    Gulf Aziz Building,       
+    Office Number 120,
```
Comment (raguiar9080) [inline contact.markdown:24]: I think these extra spaces here at the end were not meant to be added. ✂️

File: _layouts/simple.html [hunk 2]
```diff
@@ -22,12 +22,33 @@ <h2>Let's talk</h2>
...
+      </div>
+
+      <div class="address-wrapper addresses-section-container">
+      {% for Address in page.AddressesRow3 %}
+          <div class="">
+            {{Address.Address | markdownify }}
+          </div>
+      {% endfor %}
+      </div>
+
```
Comment (raguiar9080) [inline _layouts/simple.html:50]: can this whole bit be substituted by:
```
{% for AddressRow in [page.AddressesRow1, page.AddressesRow2, page.AddressesRow3] %}
  <div class="address-wrapper addresses-section-container">
    {% for Address in AddressRow %}
      <div class="">
        {{Address.Address | markdownify }}
      </div>
    {% endfor %}
  </div>
{% endfor %}
```

**Review by raguiar9080 — APPROVED** [review]
Left a few comments but nothing major. Nice job 👍

**Review by xDavidRTx — APPROVED** [review]

**Pull request merged**